### PR TITLE
[#23] Rename all occurrences of libgtmshr.so/libgtmutil.so to libyottadb.so/libyottadbutil.so

### PR DESCRIPTION
**Pull request by nars1**
Also for backward compatibility purposes
  a) install libgtmshr.so as a soft link to libyottadb.so
  b) Install libgtmutil.so as a soft link to libyottadbutil.so

### CODE REVIEW

File: sr_unix/dlopen_libyottadb.c [hunk 1]
```diff
@@ -58,18 +58,18 @@ int dlopen_libgtmshr(int argc, char **argv, char **envp, char *main_func)
...
 	/* RTLD_NOW - resolve immediately so we know errors sooner than later
-	 * RTLD_GLOBAL - make all exported symbols from gtmshr available for subsequent dlopen.
+	 * RTLD_GLOBAL - make all exported symbols from libyottadb.so available for subsequent dlopen.
 	 */
-	handle = dlopen(&gtmshr_file[0], (RTLD_NOW | RTLD_GLOBAL));
+	handle = dlopen(&yottadb_file[0], (RTLD_NOW | RTLD_GLOBAL));
 	if (NULL == handle)
 	{
-		FPRINTF(stderr, "%%GTM-E-DLLNOOPEN, Failed to load external dynamic library %s\n", gtmshr_file);
+		FPRINTF(stderr, "%%GTM-E-DLLNOOPEN, Failed to load external dynamic library %s\n", yottadb_file);
```
Comment (estess) [inline sr_unix/dlopen_libyottadb.c]: Leaving this as a hard-coded GTM error message?

File: sr_unix/dlopen_libyottadb.c [hunk 2]
```diff
@@ -58,18 +58,18 @@ int dlopen_libgtmshr(int argc, char **argv, char **envp, char *main_func)
...
 		if ((fptr = dlerror()) != NULL)
 			FPRINTF(stderr, "%%GTM-E-TEXT, %s\n", fptr);
```
Comment (estess) [inline sr_unix/dlopen_libyottadb.c]: and the same here?

File: sr_unix/dlopen_libyottadb.c (hunk 1 above)
Comment (nars1) [reply sr_unix/dlopen_libyottadb.c]: Good catch. Now fixed all occurrences in this module.

File: sr_unix/dlopen_libyottadb.c (hunk 2 above)
Comment (nars1) [reply sr_unix/dlopen_libyottadb.c]: This seems like a duplicate comment. In any case, all occurrences in this module have been fixed.

**Pull request merged**